packages/prometheus/PKGBUILD

```diff
@@ -1,5 +1,5 @@
 pkgname="carbonio-prometheus"
-pkgver="2.48.1"
+pkgver="3.0.0"
```

Comment:
why you bump version and skip the usage of `pkgver` as variable?
If nothing related to company practices, please avoid the usage of  custom version values for for third-parties.
You can always use the `pkgrel` to change the surrounding resources.

But anyway in this case is not required cause you pass from a 2.48.1 to a 2.49.1

Reply:
The requirement is the same reason we did it for videoserver, since we're adding funcionalities not related to the third party version, specifically consul integration, that could change even while maintaining the same version of prometheus. Since i'm not changing packaging but functionalities inside the two can no more match ( we already discussed and agreed on this on videoserver AFAIK)